exporter/loadbalancingexporter/log_exporter_test.go

```diff
@@ -334,6 +334,8 @@ func TestLogsWithoutTraceID(t *testing.T) {
 }
 
 func TestRollingUpdatesWhenConsumeLogs(t *testing.T) {
+	t.Skip("Flaky Test - See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/13331")
```

Comment:
Is this still failing? If so we need to reopen https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11116 and mention it here

Reply:
I changed that to track all they "periodical" tests.